service/src/main/java/com/ge/predix/acs/privilege/management/PrivilegeManagementException.java

```diff
@@ -62,4 +67,11 @@ public PrivilegeManagementException(final Throwable cause) {
         super(cause);
     }
 
+    public static ResponseEntity<JSONObject> handleIncorrectParamType() {
+        JSONObject response = new JSONObject();
+        response.put("error", "Bad Request");
```

Comment:
Pull out `"error"` into a utility class since it will definitely be used in the future (maybe somewhere in the `com.ge.predix.acs.privilege.management` package). Also use `HttpStatus.BAD_REQUEST.getReasonPhrase()` instead of the `Bad Request` literal.